### PR TITLE
feat(DateTimePicker): styles

### PR DESCRIPTION
**Pull request by yyanwang**
**What is the problem this PR is trying to solve?**
Apply styles of guideline.
Changes includes:
* add space between date picker and time picker
* set input width base on content

**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js [hunk 1]
```diff
@@ -60,25 +60,27 @@ export default function InputDatePicker(props) {
...
-		>
-			<FocusManager
-				divRef={containerRef}
-				onClick={handlers.onClick}
-				onFocusIn={handlers.onFocus}
-				onFocusOut={handlers.onBlur}
-				onKeyDown={event => {
-					handlers.onKeyDown(event, inputRef.current);
-				}}
+		<div className="date-picker">
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js]: Why did you add this extra div ?
I see that in another component you add a margin-right, I don't understand why

File: packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { format } from 'date-fns';
+
+const inputTextSizerStyle = {
+	fontWeight: 400,
+	fontSize: '16px',
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js]: rem please

File: packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	visibility: 'hidden',
+};
+
+const placeholderSizerStyle = {
+	...inputTextSizerStyle,
+	fontStyle: 'oblique',
+};
+
+function InputSizer({ placeholder, inputText, children }) {
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js]: Why do we need that ?
Why don't we have an input with a min width that would be big enough to contain the date ?

File: packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js (hunk 1 above)
Comment (yyanwang) [reply packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js]: I need this div with class "date-picker", so I can apply margin-right. the margin-right is for the space between date input and time input.

File: packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ function InputSizer({ placeholder, inputText, children }) {
 		<React.Fragment>
 			{children(width)}
 			<span style={style} ref={sizerRef}>
-				{text}
+				{text}&nbsp;
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js]: Please don't use that. Instead, call `children(width + EXTRA_SPACE)` where EXTRA_SPACE is a constant defined somewhere
